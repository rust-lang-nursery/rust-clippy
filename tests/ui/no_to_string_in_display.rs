#![warn(clippy::no_to_string_in_display)]

fn main() {
    // test code goes here
}
