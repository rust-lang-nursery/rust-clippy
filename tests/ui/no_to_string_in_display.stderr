error: Using to_string in fmt::Display implementation might lead to infinite recursion
  --> $DIR/no_to_string_in_display.rs:24:25
   |
LL |         write!(f, "{}", self.to_string())
   |                         ^^^^^^^^^^^^^^^^
   |
   = note: `-D clippy::no-to-string-in-display` implied by `-D warnings`

error: aborting due to previous error

